3rdparty/haskell/BUILD.old-time

```diff
@@ -0,0 +1,52 @@
+# vim: ft=python
+# By default 'old-time' fails to build on Windows and is very much inhermetic on
+# Linux/Darwin. The reasons are described here: https://github.com/FormationAI/hazel/issues/80
+#
+# In order to work around this we use a cabal-like 'package.bzl' generated on
+# Linux ('network-package.bzl') and tweak it depending on the platform.
+# * Some modules are not built on Windows (Posix-specific)
+# * -Dmingw32_HOST_OS and -Dx86_64_HOST_ARCH are set on Windows
+# * the 'unix' dependency is dropped on Windows
+# * on all platform we introduce a C dependency on the generated network
```

Comment:
```suggestion
# * on all platform we introduce a C dependency on the generated old-time
```

Reply:
I rewrote the comment so it's now accurate